RELEASE/scripts/autoscend/paths/pocket_familiars.ash

```diff
@@ -33,7 +35,7 @@ boolean pokefam_makeTeam()
 {
```

Comment:
`boolean pokefam_makeTeam()`
works via using cli_execute to call on ezandora script and then returning true regardless of what that script did.
as such this should be a void function as it does not return anything meaningful

or we should be importing the ezandora script in question and then calling its functions directly.